e2e/tests/add-to-cart.spec.ts

```diff
@@ -0,0 +1,48 @@
+const assert = require('assert');
+import { expect } from 'playwright/test';
+import { BASE_URL, PRODUCT_NAME, WOMEN_COLLECTIONS_ENDPOINT } from '../constants';
+import { test } from '../fixtures';
+
+test.describe('Add to cart', () => {
+  test.beforeEach('Navigate to the Collections page', async ({ page }) => {
+    await page.goto(WOMEN_COLLECTIONS_ENDPOINT);
+  });
+
+  test('User is able to add product to cart', async ({ page, collectionsPage, shoppingCart, productPage }) => {
+    if (!PRODUCT_NAME) {
+      assert(PRODUCT_NAME, 'PRODUCT_NAME must be defined');
+    }
```

Comment:
You don't need to test the var with an `if` statement and `assert` the variable. TypeScript understands `assert` statements and narrows. So if `PRODUCT_NAME` is not forced in the constants file `assert(PRODUCT_NAME)` will narrow the type from `string | undefined` to just `string`.

Reply:
Also, ideally we'd skip these tests with a notice, rather than failing them with an error. Can you investigate how we might do that? 

If Playwright supported something like this, it would be ideal:

```
if (!PLAYWRIGHT_PRODUCT_NAME) {
  skip('PLAYWRIGHT_PRODUCT_NAME was not defined, skipping test');
  return;
}
```

If we can't do this imperatively, similar to above, I can imagine some other approaches that use a bash script and the playwright tag support.